Copier/Copier.plugin.js

```diff
@@ -1040,7 +1058,7 @@ const buildPlugin = ([Plugin, Api]) => {
                     children: [
                         React.createElement("div", {
                             className: "copr-header",
-                            onClick: () => {
+                            action: () => {
```

Comment:
This breaks categories in the settings panel as `action` is not a native prop of react.